src/opencl_cryptosafe_fmt_plug.c

```diff
@@ -337,11 +339,9 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 	size_t *lws = local_work_size ? &local_work_size : NULL;
 	size_t gws = GET_NEXT_MULTIPLE(count, local_work_size);
 
-	*pcount *= mask_int_cand.num_int_cand;
+	count *= mask_int_cand.num_int_cand;
 
-	//printf("%s(%d) lws "Zu" gws "Zu" kidx %u k %d mult %u\n", __FUNCTION__, count, *lws, gws, key_idx, new_keys, mask_int_cand.num_int_cand);
-
-	if (new_keys || ocl_autotune_running) {
+	if (new_keys) {
```

Comment:
I think these changes would better be in their own PR, although that would need testing and I understand it could be handy for you to make them for these two formats now because you retest them now anyway.